docs/CITATION.md

```diff
@@ -0,0 +1,202 @@
+               "@id": "_:s1",
+               "@type": "CreativeWork",
+               "additionalType": "https://copilotstudio.microsoft.com/sensitivity-labels/v1",
+               "description": "Data is classified as Confidential but is NOT PROTECTED to allow access by approved NDA business partners. If a higher level of protection is needed please change the sensitivity level of the cited content.",
+               "keywords": ["EncryptedContent"],
+               "name": "Confidential\\Any User (No Protection)",
+               "pattern": {
+                  "@type": "DefinedTermSet",
+                  "inDefinedTermSet": "https://www.w3.org/TR/css-color-4/",
+                  "name": "color",
+                  "termCode": "orange"
+               }
+            }
+         },
+         "claimInterpreter": {
+            "@type": "Project",
+            "slogan": "Surfaced with Azure OpenAI",
+            "url": "https://www.microsoft.com/en-us/ai/responsible-ai"
+         },
+         "position": "3"
+      }
+   ],
+   "keywords": ["AIGeneratedContent"],
+   "type": "https://schema.org/Message",
+   "usageInfo": {
+      "@id": "_:s1"
+   }
+}
+```
+
+## Design decisions
+
+### Markdown reference style link
+
+This type of link allows multiple words/badges to reference to the same link, which is common when citing sources. It also helps readability when users are on text channels (email/SMS).
+
+According to CommonMark, Markdown engines should not render the reference style links separately. Channels which does not support citation should continue rendering links normally.
+
+Web Chat appended an additional UI component below the Markdown component. It visualizes the reference style link in a footnote fashion and render sensitivity labels and other UIs.
+
+### Markdown pure identifiers for badges
+
+Pure identifiers is a type of link identifiers which does not have a specific word linked. Instead, it use the identifier itself as a link.
+
+We are rendering pure identifiers as citation badges.
+
+Example of text without pure identifiers: `Here is the [link][1] about copilot.` would have the word "link" referencing the link indicated by `1` in the reference style link. It would render as "Here is the link about copilot." with the word "link" usually underlined or colored.
+
+Example of text With pure identifiers, `Here is the link[1] about copilot.` would have the identifier "1" added to the text and we would use this as a badge. The message would render as "Here is the link 1 about copilot."
+
+In Web Chat, the identifier "1" would be styled as a badge and may have a "open in new tab" icon and additional text alternative for accessibility.
+
+Note: some Markdown engine treats `link[1][1]` and `link[1]` synonymically.
+
+### Using `position` instead of `@id` to match links from Markdown to Claim
+
+Instead of using `@id`, we are using `position` to matchmake the link definition in Markdown to the Claim thing in the activity graph.
```

Comment:
```suggestion
We use `position` instead of `@id` to match the link definition in Markdown to the Claim thing in the activity graph. This simplifies the matching process and maintains an ordinal relationship between citations in the text and their corresponding Claims.
```